src/db/oracledb/images-dao.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash';
+
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+
+/**
+ * Queries data source for raw person data and passes it to the serializer
+ *
+ * @param {string} osuId OSU ID of person to select
+ * @returns {Promise<object>} Serialized person resource from person-serializer
+ */
+const getImageById = async (osuId) => {
+  const connection = await getConnection();
+  try {
+    const query = { osuId };
+    const { rows } = await connection.execute(contrib.getImageById(), query);
+
+    if (!rows || _.isEmpty(rows)) {
```

Comment:
We should return a [default image](https://github.com/osu-mist/persons-api/blob/master/src/main/resources/defaultImage.jpg) if image not found (but person exists) instead of a 404 since the resource should be person.